src/main/java/core/gui/theme/HOColorName.java

```diff
@@ -188,4 +188,12 @@
 	public static final String HRF_LIGHTBLUE_BG = "hrf.lightblue.bg";
 	public static final String HRF_DARKBLUE_BG = "hrf.darkblue.bg";
 	public static final String HRF_RED_BG = "hrf.red.bg";
+
+	// Player State colours
+	String FG_STANDARD = "player.state.standard";
+	String FG_TRANSFERLISTED = "player.state.transferlisted";
+	String FG_INJURED = "player.state.injured";
+	String FG_BRUISED = "player.state.bruised";
+	String FG_TWO_YELLOW_CARDS = "player.state.two-yellow-cards";
+	String FG_RED_CARD = "player.state.red-card";
```

Comment:
so color name are still used ?

Reply:
They are not used, but they are still defined in the main theme.  I kept them there in the case we ever need to use them again, but maybe this is a bad case of YAGNI?  I can remove if needs be.